src/components/Common/BrownBackground.js

```diff
@@ -0,0 +1,5 @@
+import styled from 'styled-components'
+
+export default styled.section`
+  background-color: ${({ theme }) => theme.colors.black};
+`
```

Comment:
Not a change or anything but `BrownBackground.js` and `theme.colors.black`??
Our legacy naming conventions in theme are so misleading. I would actually suggest a ticket to rename them.